test/e2e/internal/log/log.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package log implements test framework logging.
+package log
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+)
+
+func Logf(format string, a ...interface{}) {
```

Comment:
I'd be fine with putting this next to the `Byf` function in `common.go` instead of creating a new module. But this is also fine

Reply:
Yeah, I think both places are fine, too, but prefer to put the log function to a dedicated package because we may have more advanced log functions in the future.

Reply:
Indeed we could fetch more logs if needed like [here](https://github.com/oomichi/kubernetes/blob/master/test/e2e/framework/log.go#L1-L106) per need basis.